TFT/src/User/Menu/SendGcode.c

```diff
@@ -743,16 +788,21 @@ void menuTerminalWindow(void)
       cursorX = CURSOR_START_X;
       cursorY = terminalAreaRect[0].y0;
 
-      GUI_SetBkColor(TERM_BG_COLOR);
+      GUI_SetBkColor(fontSrcColor[infoSettings.terminal_color_scheme][2]);
       GUI_ClearPrect(&terminalAreaRect[0]);
 
       terminalDrawPageNumber();
+      pageHeadSrc = getLastSrc(terminalData->ptr[pageBufIndex]);
     }
 
     while (terminalBuf + lastTerminalIndex && (lastTerminalIndex != terminalData->terminalBufTail))
     {
       getCharacterInfo((uint8_t *)(terminalBuf + lastTerminalIndex), &info);
 
+      //read source identifier
```

Comment:
a space after "//" and "if".
Furthermore, the new settings should be provided also in config_rrf.ini.
Maybe SendGcode.c could be now renamed as Terminal.c (matching the menu label and the text repored in config.ini and Configuration.h)

Reply:
I agree